tests/drivers/build_all/virtio/boards/qemu_cortex_a53.overlay

```diff
@@ -18,3 +18,13 @@
 		status = "okay";
 	};
 };
+
+&virtio_mmio1 {
+	status = "okay";
+
+	virtio_net: virtio_net {
```

Comment:
[Node names](https://docs.zephyrproject.org/latest/contribute/style/devicetree.html) should use dashes instead of underscores (might want to fix `virtio_entropy` while you're at it 🫣 

```suggestion
	virtio_net: virtio-net {
```